img/percent_swapped.eps

```diff
@@ -0,0 +1,1067 @@
+%!PS-Adobe-3.0 EPSF-3.0
```

Comment:
this figure in supplement, so EPS likely not needed, but also it looks totally good... no transparency